.github/workflows/e2e-tests.yml

```diff
@@ -183,6 +185,13 @@ jobs:
         env:
           TERM: xterm
```

Comment:
```suggestion
          TERM: xterm
          DEBUG: replay:cypress:plugin:events,replay:test-utils:reporter
```

Reply:
@nemanjaglumac mind adding this debug envvar so we can take a look at the logs?

We seem to be missing test metadata, so we're suspicious that the plugin isn't working as it should. This should confirm some of those suspicions.